### ISSUE TITLE
Worked on windows 7 then after upgrading to 10 I get an error when opening and I cant reinstall

### ISSUE THREAD

**Issue by CloudyCrayons** [Franiac/TwitchLeecher#188]
error is
![image](https://user-images.githubusercontent.com/32446731/31065111-e1e69038-a710-11e7-8cf6-929688640589.png)

downloading the installer when clicking on the .exe nothing happens

**Comment by Franiac** [comment]
Well... send me that log file... I am super bad at "remote guessing"

**Comment by CloudyCrayons** [comment]
sure. I'm at school and I will upload it when I get back home. thank you for helping me with my issue

**Comment by CloudyCrayons** [comment]
[10022017_071841_438_PM_error.log](https://github.com/Franiac/TwitchLeecher/files/1350062/10022017_071841_438_PM_error.log)

here you go

**Comment by Franiac** [comment]
Seems like a Microsoft .NET Framework bug not realted to TL.

https://developercommunity.visualstudio.com/content/problem/55303/visual-studio-may-terminate-unexpectedly-when-runn.html

Try to update your Windows to the latest version or try 

https://support.microsoft.com/en-in/help/4022716/windows-10-update-kb4022716

I don't know if that helps but there is nothing I can do on my side. Microsoft messed that one up, not me :)

**Comment by CloudyCrayons** [comment]
That did not help me. I made sure I was on the latest version of windows and I could not download the update manually. 
![image](https://user-images.githubusercontent.com/32446731/31099311-8da15a48-a793-11e7-91d5-2e1eb7372cc3.png) 
the "Microsoft Update Catalog" had nothing

But I read the error file on a whim and I saw  errors about 
"Input.Stylus" 
and
"Input.PenThreadPool"

So I unplugged my graphics tablet and that fixed the issue. 
I didn't even have to restart my Computer. ?!?
I use twitch leecher SO MUCH that I wouldn't even be able to live without it. so i'm super glad I got it fixed. ty~
